docs/source/getting_started/writing_code.ipynb

```diff
@@ -7,10 +7,11 @@
    "source": [
```

Comment:
<p>This is somewhat confusing mixing <code>maap-py</code> info with use of <code>git</code> in the same page, alternating. <code>maap-py</code> should be it's own section and possibly it's own page. Note maap-py is only for Accessing data, and managing DPS jobs. </p><p> </p><p>Also neither is really about 'Writing code perhaps the title is wrong.<code>git</code> is about managing code. I would expect a section about writing code to cover things like, what are jupyter notebooks, what languages are supported, link to how to customize libraries, how to save data (then link to version control information). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yeah I agree that this page isn't set up the best it could be.  At the moment I'm trying to clean up stuff for the new ADE algorithm registration process (including some stuff about Github). Seeing as how this suggestion will alter previously released docs I think we should punt it to another PR/fix. Or would you prefer to clean it up now?

Reply:
It's ok to make another issue about this for later, but consider change the page title.

Reply:
future issue to re-org the page: https://github.com/MAAP-Project/maap-documentation/issues/279

Reply:
https://docs.maap-project.org/en/update_getting_started_for_new_ade/getting_started/writing_code.html#Helpful-Templates-while-developing-Algorithms-in-MAAP